agent/service-message-maven-extension/pom.xml

```diff
@@ -16,7 +16,7 @@
         <dependency>
             <groupId>org.apache.maven</groupId>
             <artifactId>maven-core</artifactId>
-            <version>3.6.3</version>
+            <version>3.9.9</version>
```

Comment:
This extension can be injected into builds with earlier Maven versions (although that was true when it was 3.6.3 as well). I think this is fine as long as we don't target newer Maven APIs, which seems unlikely in this extension.